packages/ui/src/1_atoms/HealthBar/HealthBar.stories.tsx

```diff
@@ -25,6 +25,37 @@ Default.args = {
   value: 200,
 };
 
+Default.argTypes = {
+  start: {
+    control: 'number',
+    description: 'The start value of the bar',
+  },
+  end: {
+    control: 'number',
+    description: 'The end value of the bar',
+  },
+  middleStart: {
+    control: 'number',
+    description: 'The start value of the middle bar',
+  },
+  middleEnd: {
+    control: 'number',
+    description: 'The end value of the middle bar',
+  },
+  value: {
+    control: 'number',
+    description: 'The value of the bar',
+  },
+  className: {
+    control: 'text',
+    description: 'The className to apply to the bar',
+  },
+  dataAttribute: {
+    control: 'text',
+    description: 'The data attribute to apply to the bar',
```

Comment:
```suggestion
    description: 'The data id to apply as HTML attribute to this component instance. This should be unique per component instance on the page',
```

Please update to the suggested, and you can also reuse this for any other dataAttribute props we have in other components (saves having to customise it each time)